executor.py

```diff
@@ -1,18 +1,36 @@
 from jina import DocumentArray, Executor, requests
 from InstructorEmbedding import INSTRUCTOR
+from typing import Dict
+import torch
 
 class InstructorEmbeddingExecutor(Executor):
     """InstructorEmbeddingExecutor embeds texts into 768-dim vectors using instructor embedding"""
-    def __init__(self, **kwargs):
+    def __init__(
+            self,
+            model_name: str = 'hkunlp/instructor-large',
+            batch_size: int = 32,
+            output_value: str = 'sentence_embedding',
+            convert_to_numpy: bool = True,
+            convert_to_tensor: bool = False,
```

Comment:
I would prefer to remove this parameter, we always use numpy as the output format.